nerfstudio/models/splatfacto.py

```diff
@@ -167,6 +167,12 @@ class SplatfactoModelConfig(ModelConfig):
     However, PLY exported with antialiased rasterize mode is not compatible with classic mode. Thus many web viewers that
     were implemented for classic mode can not render antialiased mode PLY properly without modifications.
     """
+    use_nd: bool = False
+    """Whether to rasterize nd values."""
```

Comment:
Expand on the docstrings a lot here, mention how each gaussians stores a feature vector of a certain dimension, and it gets rasterized as separate channels.